Sources/PointFree/VideoJs.swift

```diff
@@ -10,18 +10,61 @@ var videoJsHead: [ChildOf<Element.Head>] {
       rel(.stylesheet)
       ]),
     .init(script([
+      .init("onload", "videoJsLoaded()"),
       src("https://cdnjs.cloudflare.com/ajax/libs/video.js/7.2.4/video.min.js"),
       `defer`(true)
       ]))
   ]
 
   return [
     style(".vjs-subs-caps-button" % display(.none)),
-    script("window.HELP_IMPROVE_VIDEOJS = false;")
+    .init(node("script", [.text(unsafeUnencodedString("""
+window.HELP_IMPROVE_VIDEOJS = false
+
+function videoJsLoaded() {
+  video.addEventListener('webkitplaybacktargetavailabilitychanged', function(event) {
+    if (event.availability != "available") { return }
+
+    videojs("episode-video").ready(function() {
+      var existingAirplayControl = document.getElementsByClassName('vjs-airplay-control')
+      if (existingAirplayControl.length != 0) { return }
+
+      var controlBar = document.getElementsByClassName('vjs-control-bar')[0]
+      var video = document.getElementsByTagName('video')[0]
+
+      var template = document.createElement('div')
+      template.innerHTML = '\(render(airplayButton))'
+
+      controlBar.insertBefore(template.firstChild, controlBar.childNodes[controlBar.childNodes.length - 1])
```

Comment:
we create an empty element, through in some rendered html, and then insert it into the control bar before the fullscreen button. pretty hacky.